### PR TITLE
[11.0][ADD] l10n_nl_kvk + l10n_nl_openkvk: Kamer van Koophandel

### PR DESCRIPTION
**Pull request by astirpe**
This is a proposal to add the following two modules:

- `l10n_nl_kvk`: Kamer van Koophandel (KvK) API search
- `l10n_nl_openkvk`: OpenKvK API

The KvK field is already provided by the OCA module `partner_coc`. That field is visible when
the partner is a Company (flag *is_company* = true).

Depends on:
- [x] https://github.com/OCA/partner-contact/pull/611

l10n_nl_kvk
-----------------
This module integrates Odoo with the official Dutch chamber of commerce
*Kamer van Koophandel (KvK) API search* (https://www.kvk.nl).

The KvK API service allows lookups by the *Chamber Of Commerce Registration Number*
(KvK field) providing company name, street name, postcode and city. The lookups will be
triggered in the partner form views by entering a KvK field and pressing its lookup button.
The lookup works also on the company name field, providing Kvk number, street name, postcode
and city.

More info about the lookup service here: https://developers.kvk.nl/documentation

l10n_nl_openkvk
-----------------------

This module extends the `l10n_nl_kvk` module adding the possibility to
integrate the *Kamer van Koophandel (OpenKvK) API* by https://overheid.io/.

The OpenKvK API service allows lookups by the *Chamber Of Commerce Registration Number*
(KvK field) providing company name, street name, postcode and city. The lookups will be
triggered in the partner form views by entering a KvK field and pressing its lookup button.
The lookup works also on the company name field, providing Kvk number, street name, postcode
and city.

More info about the lookup service here: https://overheid.io/documentatie/openkvk

### CODE REVIEW

**Comment by RoelAdriaans** [comment]
You do a check for Dutch companies, but it's possible for an company that's not situated to be registered in the netherlands, https://www.kvk.nl/inschrijven-en-wijzigen/inschrijven-bij-de-kamer-van-koophandel/moet-ik-mijn-bedrijf-inschrijven/inschrijven-buitenlandse-bedrijven/
For example KvK 24147166 .. I believe that's not possible with this method?

**Comment by astirpe** [comment]
@RoelAdriaans well it's actually possible. At this moment the lookup button is available in both cases:

1. the Country is Netherlands
2. the Country is not set (=False)

It means that, if you have a company located outside the Netherlands and you set the country to false, the functionality will be available and the lookup will set the proper foreign address including the country.

I think it's not optimal but it's an acceptable solution. If you have better ideas i will be happy to adapt the code!

**Comment by astirpe** [comment]
Sorry I forgot to say: even if you set the country to Netherlands and then if you  click the lookup button, the country will be converted to the correct one.

**Review by yung-wang — APPROVED** [review]
Tests working fine here.

File: l10n_nl_openkvk/README.rst [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/l10n-netherlands-11-0/l10n-netherlands-11-0-l10n_nl_openkvk
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/176/11.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module axtends the `l10n_nl_kvk` module adding the possibility to
```
Comment (hbrunn) [inline l10n_nl_openkvk/README.rst]: extends

**Review by hbrunn — CHANGES_REQUESTED** [review]

**Review by hbrunn — APPROVED** [review]

**Comment by hbrunn** [comment]
do you want to squash stuff or should I merge as it is?

**Comment by astirpe** [comment]
@hbrunn squashed. Thanks!

**Pull request merged**